Gruntfile.js

```diff
@@ -132,6 +132,23 @@ module.exports = function(grunt) {
         }
     };
 
+    let webpackDevConfig = webpackConfig
+    webpackDevConfig.resolve.alias = {
```

Comment:
Probably you meant 
```js
Object.assign({}, webpackConfig, { resolve: ... });
```
This will `webpackConfig` now.

Reply:
Whoops, sorry. Yes. Corrected.